### ISSUE TITLE
Headless output no longer working

### ISSUE THREAD

**Issue by System64fumo** [hyprwm/Hyprland#7753]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

Yes

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch main at commit ca85455a8ed5dbe920a47c1a48de1dd993446481  (misc: Rename all instances of emtpy to empty (7522)).
Date: Mon Aug 26 13:25:52 2024
Tag: v0.42.0-70-gca85455a, commits: 5139

flags: (if any)


System Information:
System name: Linux
Node name: frostboard
Release: 6.11.0-rc1-baka+
Version: #19 SMP PREEMPT Wed Sep  4 19:46:33 IDT 2024


GPU information: 


os-release: NAME="Artix Linux"
PRETTY_NAME="Artix Linux"
ID=artix
BUILD_ID=rolling
ANSI_COLOR="0;36"
HOME_URL="https://www.artixlinux.org/"
DOCUMENTATION_URL="https://wiki.artixlinux.org/"
SUPPORT_URL="https://forum.artixlinux.org/"
BUG_REPORT_URL="https://bugs.artixlinux.org/"
PRIVACY_POLICY_URL="https://terms.artixlinux.org/docs/privacy-policy/"
LOGO=artixlinux-logo


plugins:
```

</details>

### Description

Headless outputs do not output anything to vnc (wayvnc)
Works fine for regular outputs however..

### How to reproduce

1: Run wayvnc on your server
2: Connect to it with wlvncc or an alternative vnc client
3: Observe it working just fine

4: Create a headless output in hyprland
5: Restart wayvnc server but this time set the output to the headless one
6: Connect to the server again
7: Observe it failing

### Crash reports, logs, images, videos

As stated above, Works fine for regular non headless outputs.
This used to work before, iirc prior to the AQ changes.

Thank you vaxry for breaking hyprland once more

**Comment by vaxerski** [comment]
can someone post a log

**Comment by System64fumo** [comment]
@vaxerski [hyprland.log](https://github.com/user-attachments/files/16987760/hyprland.log)

TLDR this is spammed over and over again:
[ERR] Failed to acquire swapchain buffer for HEADLESS-2
[ERR] renderer: couldn't beginRender()!

On an unrelated note.. Why do headless outputs start at HEADLESS-2 and not HEADLESS-1?

**Comment by fxzxmicah** [comment]
I have a different log:
```
[LOG] [AQ] Creating an Aquamarine backend!
[LOG] [AQ] [libseat] [libseat/backend/seatd.c:64] Could not connect to socket /run/seatd.sock: No such file or directory
[LOG] [AQ] [libseat] [libseat/libseat.c:76] Backend 'seatd' failed to open seat, skipping
[LOG] [AQ] [libseat] [libseat/libseat.c:76] Backend 'logind' failed to open seat, skipping
[ERR] [AQ] [libseat] [libseat/libseat.c:79] No backend was able to open a seat
[ERR] [AQ] libseat: failed to open a seat
[ERR] [AQ] Failed to open a session
[ERR] [AQ] DRM Backend failed
[LOG] [AQ] Starting the Aquamarine backend!
[LOG] [AQ] Starting the Wayland backend!
[ERR] [AQ] Wayland backend cannot start: wl_display_connect failed (is a wayland compositor running?)
[ERR] [AQ] Requested backend (wayland) could not start, enabling fallbacks
[ERR] [AQ] Implementation wayland failed, erasing.
[CRITICAL] m_pAqBackend couldn't start! This usually means aquamarine could not find a GPU or encountered some issues. Make sure you're running either on a tty or on a Wayland session, NOT an X11 one.
[CRITICAL] Critical error thrown: CBackend::create() failed!
```
This system (nspawn container) worked properly prior to the update with `WLR_BACKENDS=headless`.

**Comment by System64fumo** [comment]
Wait are you trying to run hyprland without a monitor?
I'm just creating a headless output for vnc

Hyprland is known to not work in fully headless mode that is a known issue

**Comment by fxzxmicah** [comment]
In the past, it worked.

**Comment by System64fumo** [comment]
Interesting.. But regardless this is a different issue.

**Comment by System64fumo** [comment]
Seems to be working now.

No idea what commit fixed it but oh well  :+1:
